mesonbuild/utils/universal.py

```diff
@@ -524,6 +524,10 @@ def miss_defaulting(self) -> "PerMachineDefaultable[T.Optional[_T]]":
             unfreeze.host = None
         return unfreeze
 
+    def clear(self, build: _T, host: _T) -> None:
```

Comment:
This is a really weird name for this function, since it doesn't actually clear the values, it sets them to new values. Can we call it `assign` or something like that?

Reply:
done